ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<p>Python-function-based -&gt; Python function-based</p><p> </p><p>"compiles the function to a component "</p><p>Should this be "kfp component"?</p><p> </p><p>" to which to write the compiled component."</p><p>Sentence was awkward to read.</p><p> </p><p>"After you run the cell below, you can view the generated component yaml file in your local directory."</p><p>Next, run the cell below to generate the component yaml file, in your local directory.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>